letsencrypt-win-simple/Services/PluginService.cs

```diff
@@ -102,13 +103,26 @@ public PluginService()
 
         private List<Type> GetResolvable<T>(bool allowNull = false)
         {
-            var ret = Assembly.GetExecutingAssembly()
-                        .GetTypes()
-                        .Where(type => typeof(T) != type && typeof(T).IsAssignableFrom(type) && !type.IsAbstract);
-            if (!allowNull)
+            var baseDir = AppDomain.CurrentDomain.BaseDirectory;
+            string[] extensions = { ".dll" };
+            var allFiles = Directory.EnumerateFileSystemEntries(baseDir, "*.*")

```

Comment:
Why not filter by *.dll instead of this construction?

Reply:
Good point, I'll update!